.script/jsonFileValidator.ts

```diff
@@ -1,13 +1,26 @@
+import fs from "fs";
+import { SchemaError, Validator } from 'jsonschema';
 import { runCheckOverChangedFiles } from "./utils/changedFilesValidator";
 import { ExitCode } from "./utils/exitCode";
-import fs from "fs";
 import * as logger from "./utils/logger";
 
 export async function IsValidJsonFile(filePath: string): Promise<ExitCode> {
-  JSON.parse(fs.readFileSync(filePath, "utf8"));
+  const json = JSON.parse(fs.readFileSync(filePath, "utf8")); 
+  if (filePath.endsWith('WorkbooksMetadata.json')) {
+    validateWorkbookSchema(json);
+  }
   return ExitCode.SUCCESS;
 }
 
+function validateWorkbookSchema(workbooksMetadataJson: object) {
```

Comment:
I think you can change this function to general for all schemas, by adding schema file path as parameter